src/crud.js

```diff
@@ -99,36 +108,73 @@ const showToDos = () => {
       const check = document.createElement('input');
       check.type = 'checkbox';
       check.classList.add('check');
+      check.id = `check-${index}`;
 
       const input = document.createElement('input');
       input.type = 'text';
       input.classList.add('activity');
       input.id = `activity-${index}`;
-      input.setAttribute('readonly', true);
       input.value = `${todo.description}`;
-      input.innerHTML = todo.description;
+      if (todo.completed) {
+        check.checked = true;
+        input.classList.add('completed');
+      }
 
       const edit = document.createElement('img');
       edit.setAttribute('src', dots);
       edit.classList.add('edit');
       edit.id = index;
 
       collection.appendChild(unordered);
-      collection.appendChild(liner);
       unordered.appendChild(ordered);
+      unordered.appendChild(liner);
       ordered.appendChild(check);
       ordered.appendChild(input);
       ordered.appendChild(edit);
+
+      ['focus', 'blur', 'keyup'].forEach((evt) => {
+        input.addEventListener(evt, (e) => {
+          if (evt === 'blur' || (evt === 'keyup' && e.key === 'Enter')) {
+          // It's a modify
+            const taskIndex = Number(e.target.id.split('-')[1]);
+            saveTodo(taskIndex);
+            e.target.blur();
+          }
+        });
+      });
```

Comment:
- [ ] Good job so far. I can see that you made changes based on the previous reviewer's suggestions. However, the changes you introduced may have caused some bugs in your application. **Right now, when I run the application locally, I tried to add a task and when I click `enter` the page reloaded and the task is neither displayed in the UI nor stored in the local storage.** Please visit back your function to add a task and check if it is working as before 🙏 As a result, I was not able to check whether the completed functionality is working.

Reply:
@BertrandConxy Thanks for your feedback. However, I am able to add new tasks on my local machine without any issue. Can you please try it again? I will add a video to demonstrate functionalities too

Reply:
@BertrandConxy here is a video demo of the app
https://www.loom.com/share/6ddf6015b06948108aad29c642d0f804

Reply:
Ok. I can see it. It looks like we even have a different UI. Let me review it again